js/apps/system/_admin/aardvark/APP/frontend/js/collections/foxxCollection.js

```diff
@@ -28,76 +28,25 @@
       this.sortOptions.desc = val;
     },
 
-    // Install Foxx from github repo
-    // info is expected to contain: "url" and "version"
-    installFromGithub: function (info, mount, callback, isLegacy, flag) {
-      var url = arangoHelper.databaseUrl('/_admin/aardvark/foxxes/git?mount=' + encodeURIComponent(mount));
-      if (isLegacy) {
+    install: function (mode, info, mount, options, callback) {
+      var url = arangoHelper.databaseUrl('/_admin/aardvark/foxxes/' + mode + '?mount=' + encodeURIComponent(mount));
+      if (options.legacy) {
         url += '&legacy=true';
       }
-      if (flag !== undefined) {
-        if (flag) {
-          url += '&replace=true';
-        } else {
-          url += '&upgrade=true';
-        }
+      if (options.setup === true) {
```

Comment:
We only want to set these if they were explicitly set. Not every install dialog has a "teardown" checkbox. This is a bit redundant for "setup" but consistency wins.